bundle/camunda-saas-bundle/src/main/java/io/camunda/connector/runtime/saas/security/SecurityConfiguration.java

```diff
@@ -43,27 +43,32 @@ public class SecurityConfiguration {
   private String issuer;
 
   @Bean
-  public WebSecurityCustomizer webSecurityCustomizer() {
-    return (web) ->
-        web.ignoring()
-            .requestMatchers(HttpMethod.POST, "/inbound/*")
-            .requestMatchers(HttpMethod.GET, "/inbound/*")
-            .requestMatchers(HttpMethod.PUT, "/inbound/*")
-            .requestMatchers(HttpMethod.DELETE, "/inbound/*")
-            .requestMatchers("/actuator/**");
-  }
-
-  @Bean
+  @Order(1)
   public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
     http.csrf(csrf -> csrf.ignoringRequestMatchers("/inbound/**"))
         .authorizeHttpRequests(
-            auth -> {
-              auth.requestMatchers("/inbound", "/tenants/**").hasAuthority("SCOPE_inbound:read");
-            })
+            auth ->
+                auth.requestMatchers("/inbound", "/tenants/**").hasAuthority("SCOPE_inbound:read"))
         .oauth2ResourceServer(oauth2 -> oauth2.jwt(jwt -> jwt.decoder(jwtDecoder())));
     return http.build();
   }
 
+  @Bean
+  @Order(0)
+  public SecurityFilterChain filterChain2(HttpSecurity http) throws Exception {
+    return http.csrf(csrf -> csrf.ignoringRequestMatchers("/inbound/**"))
+        .securityMatchers(
+            requestMatcherConfigurer ->
+                requestMatcherConfigurer
+                    .requestMatchers(HttpMethod.GET, "/inbound/*")
+                    .requestMatchers(HttpMethod.POST, "/inbound/*")
+                    .requestMatchers(HttpMethod.PUT, "/inbound/*")
+                    .requestMatchers(HttpMethod.DELETE, "/inbound/*")
+                    .requestMatchers("actuator/**"))
+        .authorizeHttpRequests(auth -> auth.anyRequest().permitAll())
+        .build();
```

Comment:
Great @mathias-vandaele, could you please document in a comment why we need a second filter chain and why the ordering is needed. It can be deducted but its not obvious. Also please move the method with the lower order upward in the code so it reflects the priority.